src/Process/Pool/Logger/Message/Builder/Factory.php

```diff
@@ -0,0 +1,17 @@
+<?php
+declare(strict_types=1);
+
+namespace Neighborhoods\Kojo\Process\Pool\Logger\Message\Builder;
+
+use Neighborhoods\Kojo\Process\Pool\Logger\Message\BuilderInterface;
+
+/** @codeCoverageIgnore */
+class Factory implements FactoryInterface
+{
+    use AwareTrait;
+
+    public function create() : BuilderInterface
+    {
+        return clone $this->getProcessPoolLoggerMessageBuilder();
```

Comment:
Per my first comment, this will shallow clone the `job`, which could lead to a lot of weirdness